packages/picasso/src/FileInput/test.tsx

```diff
@@ -13,4 +14,10 @@ describe('FileInput', () => {
 
     expect(container).toMatchSnapshot()
   })
+
+  it('can change label', () => {
+    const { container } = render(<TestFileInput buttonLabel='Upload File' />)
+
+    expect(container).toMatchSnapshot()
```

Comment:
I'd throw in a test for the label as well:
```suggestion
    expect(container).toMatchSnapshot()
    expect(getByText('Upload File')).toBeInTheDocument()
```

Reply:
Good call. Thanks!

Reply:
I would probably even remove `toMatchSnapshot`, because we already have a test with a snapshot, no need to check twice looks like if we already checking label explicitly